zipline/pipeline/factors/technical.py

```diff
@@ -683,3 +679,115 @@ def compute(self, today, assets, out, highs, lows, closes):
             )),
             2
         )
+
+
+class MovingAverageConvergenceDivergence(_ExponentialWeightedFactor):
```

Comment:
`self.weights()` is used for `calculate_ewma`, just realized it's a static method